packages/components/CHANGELOG.md

```diff
@@ -4,4 +4,4 @@
 
 ### Minor Changes
 
-- [#1731](https://github.com/Frontify/fondue/pull/1731) [`ef0519d`](https://github.com/Frontify/fondue/commit/ef0519d8021204b280b65812c98686b4ce3b909e) Thanks [@noahwaldner](https://github.com/noahwaldner)! - added export for the new components in fondue
+-   [#1731](https://github.com/Frontify/fondue/pull/1731) [`ef0519d`](https://github.com/Frontify/fondue/commit/ef0519d8021204b280b65812c98686b4ce3b909e) Thanks [@noahwaldner](https://github.com/noahwaldner)! - added export for the new components in fondue
```

Comment:
There could be a conflict between `prettier` and `markdown` generation here where we do automatic releases.

Should we ignore `CHANGELOG.md` or run a `prettier write` as part of the release creation job? cc @noahwaldner @SamuelAlev

Reply:
I added an exclusion for `CHANGELOG.md` in `.prettierignore`.

Reply:
Don't we need to revert this file?